### ISSUE TITLE
[FE] Build static pages from basic markup-language files (yaml/md)

### ISSUE THREAD

**Issue by ioggstream** [teamdigitale/dati-semantic-frontend#115]
## Comportamento atteso

- Static pages are built from markup-language (yaml/md) 
- markup should be sanitized and rendered 

## Comportamento attuale

They are embedded in [react pages](https://github.com/teamdigitale/dati-semantic-frontend/blob/main/src/components/static-content/faq/FaqBody/FaqBody.js)

## Possibili soluzioni

- See https://github.com/italia/developers.italia.it/blob/master/_data/faq.yml

**Comment by ronytw** [comment]
Just for the record, for whoever picks up this issue: we've tried the markdown approach. 
The issues we encountered were:

- The generated HTML had correct header tags (`<h1>`, `<h2>`...), but they were missing some css classes to comply with the look and feel of the site. These can be overcome by either improving the css resources, or by tweaking the Markdown to HTML conversion.
- Some mock-ups expected some behaviour which we could not easily achieve with Markdown (e.g. collapsable accordions), even though this doesn't mean that it cannot be done.
